app/client/src/sagas/InitSagas.ts

```
@@ -535,6 +537,16 @@ function* eagerPageInitSaga() {
   } catch (e) {}
 }
 
+function* registerRemainingWidgetsAndRenderAppViewerPage() {
+  yield call(registerAllWidgets);
+  yield put({ type: ReduxActionTypes.RENDER_PAGE });
+}
+
+function* handleWidgetInitSuccess() {
+  //every time a widget is initialized, we clear the cache so that all widgetFactory values are recomputed
+  clearAllWidgetFactoryCache();
+}
```

Biome (1.9.4)

[error] This generator function doesn't contain yield.



(lint/correctness/useYield)